MOBILE_API.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="SmartDeviceLink RAPI" version="7.1.0" minVersion="1.0" date="2021-04-14">
+<interface name="SmartDeviceLink RAPI" version="8.0.0" minVersion="1.0" date="2021-04-14">
```

Comment:
```suggestion
<interface name="SmartDeviceLink RAPI" version="8.0.0" minVersion="1.0" date="2021-10-27">
```

Tentatively set to prospected release date

Reply:
@JackLivio fixed in https://github.com/smartdevicelink/rpc_spec/pull/296/commits/6f6829fcb03aa353174c6f066042e5b01d495a3c